docs/pages/divider.js

```diff
@@ -1,64 +1,385 @@
 // @flow strict
 import type { Node } from 'react';
-import PropTable from '../components/PropTable.js';
 import PageHeader from '../components/PageHeader.js';
-import CardPage from '../components/CardPage.js';
 import MainSection from '../components/MainSection.js';
+import Page from '../components/Page.js';
 
-const cards: Array<Node> = [];
-const card = (c) => cards.push(c);
+export default function DividerPage(): Node {
+  return (
+    <Page title="Divider">
+      <PageHeader
+        name="Divider"
+        description="A light gray 1px horizontal or vertical line which groups and divides content in lists and layouts."
```

Comment:
ah, missed this previously: this should live as a comment in the component file and get used here as the generated version (`generatedDocGen?.description`), so users see the description on hover in their IDE